README.rst

```diff
@@ -39,7 +39,7 @@ reading and following our `Code of Conduct <code-of-conduct.md>`__.
 
 This is a ``1.9.x`` series release of the Open Data Cube.  If you are migrating from a ``1.8.x``
 series release, please refer to the
-`1.8.x to 1.9.x Migration Notes <https://datacube-core.readthedocs.io/en/latest/MIGRATION-1.8-to-1.9.html>`_.
+`1.8.x to 1.9.x Migration Notes <https://opendatacube.readthedocs.io/en/latest/MIGRATION-1.8-to-1.9.html>`_.
```

Comment:
This link doesn't work, should be to https://opendatacube.readthedocs.io/en/latest/installation/MIGRATION-1.8-to-1.9.html